src/Tablebot/Plugin/SmartCommand.hs

```diff
@@ -0,0 +1,46 @@
+-- |
+-- Module      : Tablebot.Plugin.SmartCommand
```

Comment:
Sorry for stealing the name; happy to change on request, as well as adjust the description stuff